website/server/libs/cron.js

```diff
@@ -66,20 +66,37 @@ function grantEndOfTheMonthPerks (user, now) {
 
   if (elapsedMonths > 0) {
     plan.dateUpdated = now;
-    // For every month, inc their "consecutive months" counter. Give perks based on consecutive blocks
-    // If they already got perks for those blocks (eg, 6mo subscription, subscription gifts, etc) - then dec the offset until it hits 0
-    _.defaults(plan.consecutive, {count: 0, offset: 0, trinkets: 0, gemCapExtra: 0});
-
-    for (let i = 0; i < elapsedMonths; i++) {
-      plan.consecutive.count++;
-
-      if (plan.consecutive.offset > 1) {
-        plan.consecutive.offset--;
-      } else if (plan.consecutive.count % 3 === 0) { // every 3 months
-        if (plan.consecutive.offset === 1) plan.consecutive.offset--;
-        plan.consecutive.trinkets++;
-        plan.consecutive.gemCapExtra += 5;
-        if (plan.consecutive.gemCapExtra > 25) plan.consecutive.gemCapExtra = 25; // cap it at 50 (hard 25 limit + extra 25)
+
+    let dateCreatedMoment = moment(plan.dateCreated).startOf('month');
```

Comment:
Based on @Alys' comment, I thought we couldn't update plan.dateCreated whenever a new subscription type was chosen. I.e, if you guys used that data for some type of calculation or info elsewhere. If there's no issue with me updating it whenever a new subscription is made, then I would propose it as part of this PR. 
E.g: 
User initially has a 3mo subscription, dateCreated is set for start of the 3mo sub. 
User then changes to a 6mo subscription, dateCreated gets updated for the start of the 6mo sub.